demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -43,6 +50,53 @@
 XSOAR_MARKETPLACE_FOR_IN_PACK = [ALL_MARKETPLACES_FOR_IN_PACK[0]]
 
 
+# Create a new content item with 3 commands with unique names. all commands have only 1 argument except the third command which has 2 arguments.
+generic_integration_with_3_commands_and_4_args = create_integration_object(
+    paths=["script.commands"],
+    values=[
+        [
+            {
+                "name": "command_1",
+                "description": "test",
+                "arguments": [
+                    {
+                        "name": "arg_1_command_1",
+                        "description": "nothing description.",
+                    }
+                ],
+                "outputs": [],
+            },
+            {
+                "name": "command_2",
+                "description": "test",
+                "arguments": [
+                    {
+                        "name": "arg_1_command_2",
+                        "description": "nothing description.",
+                    }
+                ],
+                "outputs": [],
+            },
+            {
+                "name": "command_3",
+                "description": "test",
+                "arguments": [
+                    {
+                        "name": "arg_1_command_3",
+                        "description": "nothing description.",
+                    },
+                    {
+                        "name": "arg_2_command_3",
+                        "description": "nothing description.",
+                    },
+                ],
+                "outputs": [],
+            },
+        ]
+    ],
+)
+
+
```

Comment:
Please move it to the parametrize.
It's a bad practice to init it as a constant from outside the tests.